### PR TITLE
add new RSA keys to relevant docs

### PR DESCRIPTION
**Pull request by equijano21**
# Description

New RSA keys mentioned in relevant doc pages.

## How has this been tested?

Local docs build.

### CODE REVIEW

File: Yubico.YubiKey/docs/users-manual/application-piv/apdu/auth-decrypt.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ limitations under the License. -->
 |:---:|:---:|:-----------:|:-------------:|:----------:|:-------------------------:|:--------:| 
 | 00  | 87  | *algorithm* | *slot number* | *data len* | *encoded data to decrypt* | (absent) |
 
-The *algorithm* is either `06` (RSA-1048) or `07` (RSA-2048). Note that it is not possible
+The *algorithm* is either `06` (RSA-1048), `07` (RSA-2048), `05` (RSA 3072), or `16` (RSA 4096). Note that it is not possible
```
Comment (equijano21) [inline Yubico.YubiKey/docs/users-manual/application-piv/apdu/auth-decrypt.md:23]: These were the values that I found using the opensc tool. Feel free to double-check these yourself if it doesn't look correct.

File: Yubico.YubiKey/src/Yubico/YubiKey/Piv/Commands/GetMetadataCommand.cs [hunk 2]
```diff
@@ -41,7 +41,7 @@ namespace Yubico.YubiKey.Piv.Commands
     ///   GetMetadataResponse metadataResponse = connection.SendCommand(metadataCommand);<br/>
     ///   if (metadataResponse.Status == ResponseStatus.Success)
     ///   {
-    ///       PivKeyMetadata keyData = metadataResponse.GetData();
+    ///       PivMetadata keyData = metadataResponse.GetData();
```
Comment (equijano21) [inline Yubico.YubiKey/src/Yubico/YubiKey/Piv/Commands/GetMetadataCommand.cs:44]: "PivKeyMetadata" didn't work when I tested this code, but "PivMetadata" did.

Comment (DennisDyallo) [reply Yubico.YubiKey/src/Yubico/YubiKey/Piv/Commands/GetMetadataCommand.cs:44]: Correct

File: Yubico.YubiKey/docs/users-manual/application-piv/apdu/auth-decrypt.md (hunk 1 above)
Comment (DennisDyallo) [reply Yubico.YubiKey/docs/users-manual/application-piv/apdu/auth-decrypt.md:23]: Correct according to our code as well

**Review by DennisDyallo — APPROVED** [review]
Great @equijano21 
There were quite a few references to the keys! Hopefully this will have been all of them. Cheers!

**Pull request merged**